cli/error_posix.go

```diff
@@ -8,7 +8,7 @@ import (
 
 func isErrnoNotSupported(err error) bool {
 	switch err {
-	case syscall.EINVAL, syscall.ENOTSUP:
+	case syscall.EINVAL, syscall.ENOTSUP, syscall.ENOTTY:
```

Comment:
It's called ENOTTY for historical reasons. It means "this ioctl is invalid for this file descriptor".